### ISSUE TITLE
Impossible to view Indicators linked to an Incident

### ISSUE THREAD

**Issue by Lhorus6** [LuatixHQ/opencti#5275]
## Description

Impossible to view Indicators linked to an Incident. **Same for all "Locations" entities** (e.g. Country, City, ...).

Usually, we have to go on the knowledge tab to do this, but for these entities, the knowledge tab doesn't offer an "Indicator" category (see screenshot)

## Environment

OCTI 5.12.9

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Create an Incident
2. Go on to Indicator and created a relation between these two Indicators and the newly created Incident
3. Go back to the Incident and try to find a way to list the Indicators linked to the Incident

## Screenshots

<img width="1567" alt="Screenshot 2023-12-20 194304" src="https://github.com/OpenCTI-Platform/opencti/assets/70956706/c36bd9dd-58a2-4928-ac7c-e74de869fb81">

**Comment by nino-filigran** [comment]
The use case seem valid, but not sure if it's a bug or rather a feature request: indeed, it seems that this screen has not been implemented. Do you have more info on this @Jipegien ? We would need anyway to tackle it imho, unless there was a specific reason not to do it.

**Comment by Jipegien** [comment]
Yes we need to implement the Indicator view on the Incident/Knowledge. Use case is already identified in the usage of EDR connectors and there is a STIX SRO dedicated to this relation ("indicates")

Regarding Locations, I don't see the use case and there is no STIX SRO dedicated to it, so it is not something urgent.

**Comment by nino-filigran** [comment]
Now possible with the "All" View.
